templates/index.html

```diff
@@ -116,7 +116,9 @@
         <div class="d1">
         <span style="color: #c9d1d9;">
             <h1>MARLon</h1>
-            <h2><span id="attackerID" style="color:rgb(126, 17, 17);">Attacker ID: XXXX</span> &emsp; <span style="color: blue;">Defender ID: XXXX</span> &emsp; Current Score: <span id="spnScore">-</span></h2>
+            <h2><span style="color:rgb(126, 17, 17);">Attacker ID: <span id="attackerID"/></span>&emsp; 
```

Comment:
```suggestion
            <h2><span style="color:rgb(126, 17, 17);">Attacker ID: {{attacker_name}} &emsp; 
```
Would this work instead of the javascript added below?

Reply:
Yes it does